cmd/manager/exec/options.go

```diff
@@ -39,7 +41,9 @@ var Options = SubscriptionCMDOptions{
 	MetricsAddr:          "",
 	KubeConfig:           "",
 	SyncInterval:         60,
-	LeaseDurationSeconds: 60,
+	LeaseDurationSeconds: 137,
```

Comment:
The existing LeaseDurationSecond option is for creating/refreshing a lease CR every minute to let cluster lifecycle knows the subscription controller is active.  This need to be remained.

I would suggest to add a new option called `LeaderElectionLeaseDurationSeconds`. This is for specifying the new leader election lease duration setting.

Reply:
Thank you for pointing this out, I've put back the old LeaseDuration and added the new option as `LeaderElectionLeaseDurationSeconds`.